tests/e2e/main.spec.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../../server/server');
+const Nightmare = require('nightmare');
+
+let nightmare;
+
+chai.use(chaiHttp);
+const expect = chai.expect;
+
+const path = 'http://localhost:8080' || process.env.PORT;
```

Comment:
This string will always be truthy, so it will never take process.env.PORT (which isn't needed in your test). Change this line to just the string